packages/currency/src/native.ts

```diff
@@ -147,6 +147,18 @@ export const nativeCurrencies: Record<RequestLogicTypes.CURRENCY.ETH, NativeEthC
       name: 'Core',
       network: 'core',
     },
+    {
+      symbol: 'ETH',
```

Comment:
ETH is the native token? It doesn't have a specific name?

Reply:
They call it ETH yes: here's some transactions:  https://explorer.zksync.io/block/19552441
Because you do have to Bridge actual ETH from mainnet.
And they reference their testnet as Goerli in a few places.

And coinmarketcap doesn't have much info on zksync naming either: https://coinmarketcap.com/currencies/zksync/

Reply:
This is causing problems, we don't call any other network's token ETH, only mainnet. 
I'll rename to ETH-zksync